janus_core/calculations/geom_opt.py

```diff
@@ -95,8 +92,7 @@ class GeomOpt(BaseCalculation):
 
     def __init__(
         self,
-        struct: Atoms | None = None,
-        struct_path: PathLike | None = None,
+        struct: Atoms | PathLike,
```

Comment:
as above

Reply:
This also only works for a single structure (also true for the phonons/EoS/geom opt: you can give it a trajectory file, but it only reads one, whereas descriptors and single point read the whole file by default).